src/xtgeo/common/sys.py

```diff
@@ -218,10 +217,12 @@ def resolve_alias(self, obj):
             obj (XTGeo instance): Instance of e.g. RegularSurface()
 
         Example::
-            >>> surf = xtgeo.RegularSurface("somefile")
-            >>> xx = _XTGeoFile("/tmp/$md5sum.gri")
-            >>> print(xx)
-            >>> '/tmp/41b24b05be153a09db6ade2f53acd6b5.gri'
+            >>> import xtgeo
+            >>> my_surface_file = surface_dir + "/topreek_rota.gri"
+            >>> surf = xtgeo.RegularSurface(my_surface_file)
+            >>> xx = _XTGeoFile("/tmp/$md5sum.gri", "rb", surf)
```

Comment:
Use `surf = xtgeo.surface_from_file(...`.

Is mdsum "safe" (will it give the same on all platforms)?

Reply:
I guess its not critical that it works on all platforms if we only doctest on linux, but I would assume that it does. Changed to `surface_from_file`.